WordPress/Classes/Utility/iAds/SearchAdsAttribution.swift

```diff
@@ -0,0 +1,123 @@
+/// Implementation of Search Ads attribution details
+/// More info: https://searchads.apple.com/help/measure-results/#attribution-api
+
+import Foundation
+import iAd
+import AutomatticTracks
+
+@objc final class SearchAdsAttribution: NSObject {
+
+    /// Keep the instance alive
+    ///
+    private static var lifeToken: SearchAdsAttribution?
```

Comment:
One note: can we perhaps rename this to `instance` (or something that makes it extra clear it's a pointer to the current instance?)